src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/MSBuildSdkExtrasProject.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.IO;
+
+namespace Xamarin.ProjectTools
+{
+	public class MSBuildSdkExtrasProject : DotNetStandard
+	{
+		public MSBuildSdkExtrasProject ()
+		{
+			Sdk = "MSBuild.Sdk.Extras/2.0.41";
+			Sources.Add (new BuildItem.Source ("Class1.cs") {
+				TextContent = () => "public class Class1 { }"
+			});
+			Sources.Add (new AndroidItem.AndroidResource ("Resources\\values\\Strings.xml") {
+				TextContent = () => StringsXml.Replace ("${PROJECT_NAME}", ProjectName)
+			});
+			StringsXml = XamarinAndroidLibraryProject.default_strings_xml;
+			TargetFrameworks = "MonoAndroid10.0";
```

Comment:
Note that I had to use `MonoAndroid10.0`, since `MonoAndroid100` set the TFV to `v1.0.0`...

However, `MonoAndroid90` works property: TFV is `v9.0`.

Reply:
Is that going to cause us problems? Or will this just affect those using MSBuild.Sdk.Extras?

Reply:
I have a feeling that NuGet authors will have to use `monoandroid10.0` for their directory names... where they have been used to using `monoandroid81`, `monoandroid90`, etc.